### ISSUE TITLE
Highlight single entry tables correctly

### ISSUE THREAD

**Issue by UlrichKu** [chameleon-system/chameleon-system#478]
**Problem**

Accessing a table with only one entry (e. g. portal settings with the option "Table contains only one record" on): There will be no title, breadcrumb or highlight in the sidebar that hints what page might be active.

Moreover the last section in the sidebar that was accessed before will be expanded.

This is the view for "portal settings":
![Selection_165](https://user-images.githubusercontent.com/20949224/63146983-1024f780-bffd-11e9-9861-ba0497e8f653.png)

**Describe the solution you'd like**
Also single-entry tables should have the proper breadcrumb. The correct sidebar section should be expanded.

**Comment by pixeljunkie** [comment]
I think there is no simple solution for this. There IS no table accessible, so we can't add the table name to the history.
Or maybe we could... but we should then link the breadcrumb item of the table name to the record, too or add the item without a link target.

**Comment by UlrichKu** [comment]
Current PR https://github.com/chameleon-system/chameleon-base/pull/472 now addresses this rather broadly with introducing a new backend breadcrumb module which should be showing a sensible path in most situations.

**Comment by pixeljunkie** [comment]
Something relevant popped up which has to to with the changes done here, so should be checked, too: #651

**Comment by UlrichKu** [comment]
A more proper solution here could be to simply make the current history tab-aware. So each browser tab has its own history (breadcrumb).

**Comment by bestform** [comment]
Yes, we should discuss this further. I will move this issue to the next milestone for this.
